File: __tmp_edit.js
const fs = require('fs');
const path = 'embyClient.js';
const contents = fs.readFileSync(path, 'utf8');
const lines = contents.split(/\r?\n/);
const result = [];
for (let i = 0; i < lines.length; i++) {
  const line = lines[i];
  if (line.includes('${view.Id}::favorites')) {
    i += 3; // skip type, name, closing lines
    continue;
  }
  result.push(line);
}
fs.writeFileSync(path, result.join('\n'), 'utf8');

File: __tmp_edit.py
lines = []
import pathlib
path = pathlib.Path('embyClient.js')
with path.open('r', encoding='utf-8') as f:
    for line in f:
        lines.append(line)
result = []
skip = 0
for line in lines:
    if skip > 0:
        skip -= 1
        continue
    if '${view.Id}::favorites' in line:
        skip = 3  # skip type line, name line, closing line
        continue
    result.append(line)
path.write_text(''.join(result), encoding='utf-8')
